### ISSUE TITLE
Support listener ssl policy

### ISSUE THREAD

**Issue by softprops** [aws/copilot-cli#1342]
Aws load balancer listeners can restrict the tls protocol versions clients use to connect through an [sslpolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listener.html#cfn-elasticloadbalancingv2-listener-sslpolicy). The default policy allows for insecure protocol versions for compatibility.

For those with stricter security goals it would be great to expose the ability to only accept current tls protocol versions.

```yaml
# Your service name will be used in naming your resources like log groups, ECS services, etc.
name: frontend
# The "architecture" of the service you're running.
type: Load Balanced Web Service

image:
  # Path to your service's Dockerfile.
  build: ./Dockerfile
  # Port exposed through your container to route traffic to it.
  port: 80

http:
  # Requests to this path will be forwarded to your service. 
  # To match all requests you can use the "/" path. 
  path: '/'
  security_policy: ELBSecurityPolicy-FS-1-2-Res-2019-08

```

**Comment by neekey** [comment]
It seems like we already have a similar setting for NLB (https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#nlb-ssl-policy), I wonder why ALB does not support the `ssl_policy` feature yet

**Comment by iamhopaul123** [comment]
Hello @neekey. Yes NLB supports this feature but unfortunately for our ALB services we still haven't been able to work on it yet. Thank you for bringing in up this issue again.

**Comment by huanjani** [comment]
This feature is now released in v1.23.0: https://github.com/aws/copilot-cli/releases/tag/v1.23.0! 🎉

**Comment by neekey** [comment]
amazing! thank you @huanjani @iamhopaul123
